src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/util/SalesforceSplitUtil.java

```diff
@@ -22,11 +22,7 @@
 import com.sforce.async.JobInfo;
 import com.sforce.async.JobStateEnum;
 import com.sforce.async.OperationEnum;
-import io.cdap.plugin.salesforce.BulkAPIBatchException;
-import io.cdap.plugin.salesforce.SObjectDescriptor;
-import io.cdap.plugin.salesforce.SalesforceBulkUtil;
-import io.cdap.plugin.salesforce.SalesforceConnectionUtil;
-import io.cdap.plugin.salesforce.SalesforceQueryUtil;
+import io.cdap.plugin.salesforce.*;
```

Comment:
We should not have * imports. Import only the classes you need.
You can do a `mvn clean package` after you address any comments for sanity check, that will catch issues some issues like this for you

Reply:
Thanks. I fixed this on my latest build.

Reply:
You can turn off automatic wildcard imports - https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-